src/main/java/com/stripe/model/SubscriptionSchedule.java

```diff
@@ -0,0 +1,406 @@
+// Generated by com.stripe.generator.entity.SdkBuilder
+
+package com.stripe.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.stripe.Stripe;
+import com.stripe.exception.StripeException;
+import com.stripe.net.ApiResource;
+import com.stripe.net.RequestOptions;
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@EqualsAndHashCode(callSuper = false)
+public class SubscriptionSchedule extends ApiResource
+    implements HasId, MetadataStore<SubscriptionSchedule> {
+  /**
+   * Either `charge_automatically`, or `send_invoice`. When charging automatically, Stripe will
+   * attempt to pay the underlying subscription at the end of each billing cycle using the default
+   * source attached to the customer. When sending an invoice, Stripe will email your customer an
+   * invoice with payment instructions.
+   */
+  @SerializedName("billing")
+  String billing;
+
+  /**
+   * Define thresholds at which an invoice will be sent, and the subscription advanced to a new
+   * billing period.
+   */
+  @SerializedName("billing_thresholds")
+  Subscription.BillingThresholds billingThresholds;
```

Comment:
I like that we re-use it "magically". You likely do that with client_metadata though after our talk today I still really hope that it's the same resource in pay-server and so we should be able to know that automatically.